### PR TITLE
Shorter expand/shrink selection shortcuts on mac

### PR DESCRIPTION
**Pull request by solomatov**

### CODE REVIEW

**Comment by solomatov** [comment]
It seems that the build check on linux failed with an error unrelated to this change. What should be done about it?

File: src/vs/editor/contrib/smartSelect/smartSelect.ts [hunk 1]
```diff
@@ -165,7 +165,10 @@ class GrowSelectionAction extends AbstractSmartSelect {
 			kbOpts: {
 				kbExpr: EditorContextKeys.editorTextFocus,
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.RightArrow,
-				mac: { primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow },
+				mac: {
+					primary: KeyMod.CtrlCmd | KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow,
+					secondary: [KeyMod.WinCtrl | KeyMod.Shift | KeyCode.RightArrow],
+				},
```
Comment (jrieken) [inline src/vs/editor/contrib/smartSelect/smartSelect.ts]: This is only for mac, right? How about the other platforms?

**Review by jrieken — CHANGES_REQUESTED** [review]

File: src/vs/editor/contrib/smartSelect/smartSelect.ts (hunk 1 above)
Comment (solomatov) [reply src/vs/editor/contrib/smartSelect/smartSelect.ts]: On other platforms, they are good enough: the primary shortcut is KeyMod.Shift | KeyMod.Alt | KeyCode.LeftArrow, which is the same length as Mac.

Comment (jrieken) [reply src/vs/editor/contrib/smartSelect/smartSelect.ts]: Ah, makes sense. Thanks for clarifying

**Review by jrieken — APPROVED** [review]

**Pull request merged**